src/app/forms/custom-sections/dimensions/dimensions.component.ts

```diff
@@ -66,11 +71,19 @@ export class DimensionsComponent implements OnInit, OnChanges, OnDestroy {
     return FormNodeEditTypes;
   }
 
-  get dimensions(): FormGroup {
-    return this.form.get(['dimensions']) as FormGroup;
+  get dimensions(): CustomFormGroup {
+    return this.form.get(['dimensions']) as CustomFormGroup;
+  }
+
+  get hasAxleSpacings(): boolean {
+    return !!this.vehicleTechRecord.dimensions?.axleSpacing?.length;
+  }
+
+  get axleSpacings(): CustomFormGroup {
+    return this.form.get(['dimensions', 'axleSpacing']) as CustomFormGroup;
   }
 
-  get bottomSection(): CustomFormGroup {
-    return this.form.get(['dimensionsBottomSection', 'dimensions']) as CustomFormGroup;
+  getAxleSpacing(i: number): CustomFormGroup {
```

Comment:
I know you're passing in I as an index from a different place in the code, but could this I have a better name ?
index would do

Reply:
This same pattern is used in other places too. `index` is too verbose man 🙂